packaging_automation/publish_docker.py

```diff
@@ -170,9 +181,13 @@ def publish_tagged_docker_images(docker_image_type, tag_name: str, will_image_be
 def publish_nightly_docker_image(will_image_be_published: bool):
     print("Building nightly image...")
     docker_image_name = f"{DOCKER_IMAGE_NAME}:{docker_image_info_dict[DockerImageType.nightly]['docker-tag']}"
-    docker_client.images.build(dockerfile=docker_image_info_dict[DockerImageType.nightly]['file-name'],
-                               tag=docker_image_name,
-                               path=".")
+    _, logs = docker_client.images.build(dockerfile=docker_image_info_dict[DockerImageType.nightly]['file-name'],
+                                         tag=docker_image_name,
+                                         path=".")
+    for log in logs:
+        log_str = log.get("stream")
+        if log_str:
+            print(log_str, end='')
```

Comment:
Before merging:
- [ ] please move this into a function and call from those three places

```python
def flush_docker_logs(json_decoded_docker_logs):
    for log in json_decoded_docker_logs:
        log_str = log.get("stream")
        if log_str:
            print(log_str, end='')
```